src/pages/[...slug].vue

```diff
@@ -0,0 +1,5 @@
+<template>
+  <section class="default-layout__info default-layout__info--info-page">
+    <h1>Page not found</h1>
```

Comment:
Can we localise this?

Reply:
This exact file is gone. The new error page is currently not localisable, cuz the locale kinda comes from the URL, which can be invalid enough to not include any supported locale. We can however do it using just the browser preferences. Sounds like a separate little PR!